src/logic/operate.test.js

```diff
@@ -0,0 +1,5 @@
+import operate from './operate';
+
+test('Summation of 2 numbers', () => {
+  expect(operate(2, 3, '+')).toBe('5');
+});
```

Comment:
- [ ] It is important to test all functionalities to ensure there is no bug in your code. Please add more tests for operate.js, ensure it has individual for tests all the functionalities and scenarios.

Reply:
Hi @Ghiftee ,  are those all the changes i need to fix ? Otherwise i have fixed the issue and ready to push.
Thank you. 😄

Reply:
Yes they are @billionsjoel. If you have fixed them then you're good to go 👍🏽